solutions/only_ints.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+A module for verifying integer inputs.
+
+Module contents:
+    - only_ints: A function to check if two parameters are both integers.
+
+Created on 12 29 2024
+@author: Mohammad
```

Comment:
Doctests are missing (at least three passing doctests)

Reply:
I almost forgot about it—this can be added to the code for better functionality.